### PR TITLE
Style

### PR DESCRIPTION
**Pull request by brandones**

### CODE REVIEW

**Comment by brandones** [comment]
Screenshot from 2020-04-02 11-15-51

Screenshot from 2020-04-02 11-19-55

File: src/referrals-queue/referrals-queue.component.tsx [hunk 1]
```diff
@@ -31,17 +31,20 @@ export default function ReferralsQueue(props: ReferralsQueueProps) {
...
+        <div className="omrs-type-title-2">
           <Trans i18nKey="referrals-queue">Referrals Queue</Trans>
         </div>
-        <div>
-          <div className="omrs-input-group">
-            <label>
-              <Trans i18nKey="from">From</Trans>
+        <div className={styles.controlsContainer}>
+          <div className={`${styles.inputContainer}`}>
+            <div className={styles.dateInputContainer}>
```
Comment (mogoodrich) [inline src/referrals-queue/referrals-queue.component.tsx:41]: what's with the nested brackets and the quotes and the $ on line 40?

**Review by mogoodrich — APPROVED** [review]
Added  a couple y questions but generally LGTM (though my knowledge of CSS and CSS best practices is rather limited).

As for the actual design itself, probably to best to get as many of us looking at it as possible so we can debate what kind of look we want to go for.

File: src/referrals-queue/referrals-queue.component.tsx (hunk 1 above)
Comment (brandones) [reply src/referrals-queue/referrals-queue.component.tsx:41]: Oops, cruft. That syntax is useful if you want to use dynamic and static class names together. The backticks are ES6 string interpolation. In brief, the syntaxes are:

```
className="some-static-name"
className={styles.anImportedClass}
className={`some-static-name ${styles.anImportedClass}`}
```

**Pull request merged**